composer.json

```diff
@@ -10,11 +10,9 @@
 		"rullzer/easytotp": "^0.1.4"
 	},
 	"require-dev": {
+		"bamarni/composer-bin-plugin": "^1.8",
```

Comment:
for other apps like Mail I had to push this into the regular requires, not dev because it's need to run `composer i --no-dev` successfully while packaging the app

this app is shipped, so it might not be necessary. but please test

Reply:
A bin script can be added as a fallback to `composer.json` and then it works.

Ref https://github.com/bamarni/composer-bin-plugin#auto-installation.

Reply:
Does it work, though? Last time I tried the forwarding with 1.x it didn't actually forward

Reply:
Yes it works now. I tried a clean dev setup [1] and packaging the app [2].

[1] `rm -r vendor* && gco vendor-bin && composer i && composer run cs:check`
[2] `krankerl package`